doc/source/install/backend-linux-install.rst

```diff
@@ -5,7 +5,7 @@ Installation du backend sous Linux
 Pour installer une version locale de ZdS sur GNU/Linux, veuillez suivre les instructions suivantes.
 Si une commande ne passe pas, essayez de savoir pourquoi avant de continuer.
 
-Les commandes suivantes sont génériques et indépendantes de la distribution que vous utilisez.
+Certaines des commandes d'instalation (``apt-get``) sont données ici pour Debian et ces dérivés, pour lesquels il est sûr qu'elles fonctionnent. Néanmoins, si vous utilisez une distribution différente, le nom des paquets à installer devrait être fort semblable, n'hésitez dès lors pas à employer la fonction "recherche" de votre gestionnaire de paquet préféré. Les autres commandes sont génériques et indépendantes de la distribution utilisée.
```

Comment:
> Zut, faudra attendre mes 18 ans :D (dans 2 ans)

What the f\*\*\* ? En Belgique, tu peux acheter de la bière à partir de 16 ans. Anyway, ce sera donc un coca ou un jus d'orange, dans ce cas, désolé ^^

> Je ne sais pas comment changer ça dans PyCharm. Une solution est de passer ton texte à travers http://bonpatron.com (attention, il y a une limite d'environ 2000 caractères par envoi) ;)

... Et il est complètement perdu avec le formatage en markdown. Enfin c'est pas grave, je vais chercher ^^